lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -34,6 +88,7 @@ def powershell_args
   end
 
   def exists?
+    set_symlink_folder
```

Comment:
Do we cause problems by not re-symlinking each run?

For instance, if you're developing locally / testing and you happen to move source... by not resymlinking to the current directory, you'll be pointing to the wrong thing.

Not sure if there are any legit runtime scenarios where that would be a problem or not.